### ISSUE TITLE
Cochlear chondritis

### ISSUE THREAD

**Issue by carolina-f** [obophenotype/human-phenotype-ontology#11378]
**Preferred term label:**

**Synonyms**
Auricular chondritis

**Definition (free text, please give PubMed ID)**
PMID: 24626673 According to Firestein, patients with MAGIC present specific findings for BD and RP, respectively, orogenital aphthous ulcers and chondritis (**auricular**, nasal, or respiratory tract), can also present less specific manifestations such as audiovestibular dysfunction, large artery thrombosis and aneurisms, as other signs and symptoms common to many rheumatic disorders, such as ocular changes, pseudofolliculitis, migratory polyarthritis, glomerulonephritis, gastrointestinal tract involvement, and central nervous system involvement.

PMID: 32914210 Our patient also had features suggesting RP: respiratory tract chondritis (hoarseness, shortness of breath), **cochlear chondritis** (vertigo and tinnitus), and 99mTc-MDP bone scintigraphy suggested costal chondritis.
Moreover, these two clinical entities responded to GC dose increasing and received remission with immunosuppressive agents. Thus, we were able to confirm the diagnosis of MAGIC syndrome in the present case.

**Parent term (use hpo.jax.org/app)**
Chondritis HP:0100662

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
MAGIC syndrome (mouth and genital ulcer with inflamed cartilage) ORPHA:324972

**Your nano-attribution (ORCID)**
0009-0005-6714-5727

**Comment by pnrobinson** [comment]
@carolina-f  I do not think the cochlea has cartilage like this -- do you mean Chondritis of pinna HP:0200047

**Comment by carolina-f** [comment]
Finally, I think I was wrong, my bad.
I have read articles about ORPHA:728 Relapsing polychondritis in which they also mention cochlear-vestibular dysfunction with severe sensorineural hearing loss in both ears.
In PMID: 20049455 they mention: Schuknecht noted the absence of cartilage within the inner ear and implicated an obliterative vasculitis of the labyrinthine artery or its branches. This hypothesis is consistent with the known occurrence of systemic necrotic vasculitis in RP. This case met all of the criteria for RP described by McAdam. This patient also had histologic findings (the cartilage matrix loses basophilic staining, the inflammatory cells infiltrate the cartilage, chondrocytes become vacuolated and necrotic, the cartilage is replaced by fibrous tissue) consistent with an RP diagnosis. Sensorineural hearing loss appears in 40–50% of
patients with RP during the course of the disease, but most do not progress to profound hearing loss; therefore,
treatments commonly applied to RP patients such as NSAIDs, steroids, or immunosuppressive agents (AZT,
cyclophosphamide, etc.) can improve or preserve the hearing of patients with RP.

PMID: 15291945 : There are no specific tests for RPC.
In most cases, elevated erythrocyte sedimentation rate (ESR) and mild anemia are present. McAdam et al. (1976) had earlier
suggested clinical criteria for the diagnosis of the disease, which included chondritis of both auricles, non-erosive inflammatory polyarthritis, nasal chondritis, ocular inflammation, respiratory tract chondritis and cochlear and/or vestibular damage.

But although cochlear vestibulo involvement is mentioned, the term cochlear chondritis is not mentioned.

**Comment by pnrobinson** [comment]
Agree, I do not think that cochlear chondritis can exist, I will close this.
